pycqed/measurement/waveform_control/pulsar.py

```diff
@@ -1073,8 +1357,15 @@ def __init__(self, name='Pulsar', master_awg=None):
                            get_cmd=self._get_inter_element_spacing)
         self.add_parameter('reuse_waveforms', initial_value=False,
                            parameter_class=ManualParameter, vals=vals.Bool())
-        self.add_parameter('append_zeros', initial_value=0, vals=vals.Ints(),
+        self.add_parameter('use_sequence_cache', initial_value=False,
+                           parameter_class=ManualParameter, vals=vals.Bool(),
+                           set_parser=self._use_sequence_cache_parser)
+        self.add_parameter('prepend_zeros', initial_value=0, vals=vals.Ints(),
                            parameter_class=ManualParameter)
+        # keep old parameter name for backwards compatibility
+        self.add_parameter('append_zeros', initial_value=0, vals=vals.Ints(),
+                           set_cmd=(lambda v, self=self: self.prepend_zeros(v)),
+                           get_cmd=(lambda self=self: self.prepend_zeros()))
```

Comment:
Maybe we can remove the old parameter now? I guess prepending worked better in combination with segment filter.

Reply:
Yes, you could push this to the PR branch.

Reply:
Note: There will be another PR concerning pulsar soon, and we could also do it as part of this other PR. It would be good if we could move on with #210 ASAP @antsr